### DISCUSSION TITLE
是否需要云函数命名空间？

### DISCUSSION THREAD

**Discussion by skyoct** [labring/laf#170] [Generals]
导入别人云函数命名重复如何处理？比如两个插件，一个阿里云发送短信，一个腾讯云发送短信，在不同人手上开发时候可能需要命名为send_message，那么同时导入两个函数时候如何处理？
如果有命名空间概念，就不会引起冲突。

**Comment by maslow** [comment]
这个想法值得讨论：namespace 怎样命名，如何保证唯一

**Comment by object-kaz** [comment]
考虑借助目录的方式来实现命名空间：

```
tencent/send_message
aliyun/send_message
```

后面如果想把云函数导出成文件结构的话也非常方便。

**Reply by maslow** [reply]
是否需要支持多层目录/名字空间？
